### PR TITLE
Feature/add signs to doors

### PR DESCRIPTION
**Pull request by theepsi**

### CODE REVIEW

File: Source/VoidSpace/DoorPanel.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	
+}
+
+// Called every frame
+void ADoorPanel::Tick(float DeltaTime)
+{
+	Super::Tick(DeltaTime);
+
+}
+
```
Comment (polvila) [inline Source/VoidSpace/DoorPanel.cpp]: Tick method can be deleted

File: Source/VoidSpace/DoorPanel.cpp [hunk 2]
```diff
@@ -16,51 +16,30 @@ ADoorPanel::ADoorPanel()
...
-	MaterialExitVault = exitVault.Object;
-	MaterialMaintenance = maintenance.Object;
-	MaterialObservatory = observatory.Object;
-	MaterialControlRoom = controlRoom.Object;
+// Called every frame
+void ADoorPanel::Tick(float DeltaTime)
+{
+	Super::Tick(DeltaTime);
 
 }
```
Comment (polvila) [inline Source/VoidSpace/DoorPanel.cpp]: Tick method is not necessary

File: Source/VoidSpace/DoorPanel.cpp [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	static ConstructorHelpers::FObjectFinder<UStaticMesh> panelMesh(TEXT("StaticMesh'/Game/Meshes/DoorPanel/DoorsPanel.DoorsPanel'"));
+	PanelMesh = CreateDefaultSubobject<UStaticMeshComponent>(TEXT("PanelMesh"));
+
+	RootComponent = PanelMesh;
+
+	PanelMesh->SetStaticMesh(panelMesh.Object);
+}
+
+// Called every frame
+void ADoorPanel::Tick(float DeltaTime)
```
Comment (jowie94) [inline Source/VoidSpace/DoorPanel.cpp]: Tick can be deleted

File: Source/VoidSpace/DoorPanel.cpp (hunk 2 above)
Comment (theepsi) [inline Source/VoidSpace/DoorPanel.cpp]: ups, sorry just forgot to remove it, thx!

**Pull request merged**